Diggy/Feature/DiggyTilePressure.lua

```diff
@@ -0,0 +1,52 @@
+--[[-- info
+    Provides the ability to collapse caves when digging.
+]]
+
+-- dependencies
+
+local Event = require 'utils.event'
+local PressureMap = require 'Diggy.PressureMap'
+
+-- this
+local DiggyTilePressure = {}
+
+
+
+--[[--
+    Registers all event handlers.]
+
+    @param config Table {@see Diggy.Config}.
+]]
+function DiggyTilePressure.register(config)
+    Event.add(PressureMap.events.on_pressure_changed, function(event)
+      
+        local e = game.surfaces['nauvis'].find_entity("flying-text", event.position)

```

Comment:
Is it possible to update the values of existing entities? That would increase performance if we locally keep track of them. We would only have to reposition the entities and change the value instead of removing and recreating.

Reply:
Yes! I will change that.